etc/docker/elyra/README.md

```diff
@@ -16,5 +16,17 @@ limitations under the License.
 {% endcomment %}
 -->
 
-### elyra
-Builds the Elyra image for use as standalone or with JupyterHub. See [Deploying Elyra and JupyterHub in a Kubernetes environment](https://elyra.readthedocs.io/en/latest/recipes/deploying-elyra-in-a-jupyterhub-environment.html#deploying-elyra-jupyterhub-in-a-kubernetes-environment)
+### Elyra Docker Image
+
+Builds the Elyra image for use as standalone or with JupyterHub.
+
+See [Deploying Elyra and JupyterHub in a Kubernetes environment](https://elyra.readthedocs.io/en/latest/recipes/deploying-elyra-in-a-jupyterhub-environment.html#deploying-elyra-jupyterhub-in-a-kubernetes-environment)
+
+#### Building a custom container image
+
+To build a custom version of this container image:
+1. Clone this repository
+2. Build the image using the `Dockerfile` in the `kubeflow/notebooks` directory:
```

Comment:
The directory reference appears to be invalid

```suggestion
2. Build the image using the `Dockerfile` in the `etc/docker/elyra` directory:
```

Reply:
Done in https://github.com/elyra-ai/elyra/pull/2344/commits/c5e17124f04ab5d2016ceb558cda1281e146ca12